src/sonic-config-engine/portconfig.py

```diff
@@ -51,4 +70,98 @@ def parse_port_config_file(port_config_file):
             port_alias_map[data['alias']] = name
     return (ports, port_alias_map)
 
+def parse_platform_json_file(port_config_file):
+    ports = {}
+    port_alias_map = {}
+
+    # Read 'platform.json' file
+    try:
+        with open(port_config_file) as fp:
+            try:
+                data = json.load(fp)
+            except json.JSONDecodeError:
+                print("Json file does not exist")
+        port_dict = ast.literal_eval(json.dumps(data))
+    except:
+        print("error occurred while parsing json:", sys.exc_info()[1])
 
+    """ Regex for  SYMMETRIC breakout_modes
```

Comment:
Is this Regex logic would be used by "sonic-utilites" CLI too?  Is it possible to put it in a library and so portconfig.py and CLI can share?

Reply:
I am thinking to update the CLI to get these regexes from this **"proconfig.py"** file. Does it sound good?